examples/platform/silabs/efr32/rs911x/hal/sl_si91x_ncp_utility.c

```diff
@@ -107,12 +105,17 @@ sl_status_t sl_wfx_host_pre_lcd_spi_transfer(void)
  **********************************************************/
 sl_status_t sl_wfx_host_post_lcd_spi_transfer(void)
 {
-    USART_Enable(SL_MEMLCD_SPI_PERIPHERAL, usartDisable);
-    CMU_ClockEnable(SPI_CLOCK(SL_MEMLCD_SPI_PERIPHERAL_NO), false);
-    GPIO->USARTROUTE[SL_MEMLCD_SPI_PERIPHERAL_NO].ROUTEEN = PINOUT_CLEAR;
+    uint32_t status = SL_STATUS_OK;
 #if SL_SPICTRL_MUX
     xSemaphoreGive(spi_sem_sync_hdl);
 #endif // SL_SPICTRL_MUX
+    status = sl_board_disable_display();
+    if (SL_STATUS_OK != status)
+    {
+        SILABS_LOG("sl_board_disable_display failed with error: %x", status);
+        xSemaphoreGive(spi_sem_sync_hdl);
+        return status;

```

Comment:
you can remove this return here and just always `return status;` at the end on the function instead on SL_STATUS_OK